Gemfile

```diff
@@ -126,3 +121,5 @@ end
 group :test do
   gem 'haikunator', '~> 1' # for generating random names in tests
 end
+
+gem "webrick", "~> 1.7"
```

Comment:
Why are we pulling Webrick in?  If it's just for development, let's wrap it in the `:development` group.

Reply:
When possum is consumed in appliance test image, sv services won't start up due to missing webrick package.
Moved to :test :development group.